prow/plugins.yaml

```diff
@@ -84,6 +85,10 @@ plugins:
   - lifecycle
   - verify-owners
 
+  istio/operator:
+  - approve
```

Comment:
Will `/approve` work without a OWNERS file? I am not sure. An alternative would be to add a new tide query with `review: approved` as a part instead of looking for lgtm and approved

Reply:
I doubt it.  do we need an approved label for tide to merge the work?  The way I parsed the issue, lgtm is sufficient.

Reply:
@howardjohn your approach may work better than the proposal in the issue tracker.  What I've tried thus far is not passing the linting...

Reply:
https://github.com/istio/test-infra/blob/c01c3a1f196fe7901aa76e1b3d9cf68a5f482883/prow/config.yaml#L193-L206 says that for the four listed repo (and this PR adds a fifth) tide will try and merge any PR that has `approved`, `lgtm` and `cla: yes` labels. 

You can create another section with your own config (note that `istio-releases/pipeline` does this already). Or you can add an OWNERS file in operator and turn on the approve plugin to make sure we get the label added.

Reply:
The pattern we want for the operator repo is:

- CODEOWNERS and not OWNERS
- No /foo commands
- No "approve" or "lgtm" labels defined or used in the repo.
- Users approve PRs using the normal GitHub approval/comment UI.

Reply:
I understand @geeknoid :)